lib/repositories/match_details_repository.dart

```diff
@@ -165,4 +166,162 @@ class MatchDetailsRepository {
     ];
     return response.map(MatchEventsModel.fromJson).toList();
   }
+
+  Future<List<LineUpModel>> fetchMatchLineUps(String matchID) async {
+    // return (await liveMatchesRemoteService.fetchMatchLineUps(matchID: matchID)).response;
```

Comment:
Remove comments

Reply:
I left them there  in order to not overuse Api calls